src/Cdn/Cdn.Autorest/test/Test-AzFrontDoorCdnEndpointCustomDomain.Recording.json

```diff
@@ -34,7 +34,7 @@
         "Content-Type": [ "application/json; charset=utf-8" ],
         "Expires": [ "-1" ]
       },
-      "Content": "{\"customDomainValidated\":false,\"reason\":\"NotExist\",\"message\":\"Custom domain dose not exist in the profile. Please make sure the domain exsits in the profile.\"}",
+      "Content": "{\"customDomainValidated\":false,\"reason\":\"NotExist\",\"message\":\"Custom domain dose not exist in the profile. Please make sure the domain exists in the profile.\"}",
```

Comment:
No need to update recording files as they would be overwritten the next time when re-recorded.

Reply:
Ok, what generates them ? as it means the change has to be at the source ?

Reply:
reverted as requested